mathesar/rpc/tables.py

```diff
@@ -44,3 +45,25 @@ def list_(*, schema_oid: int, database_id: int, **kwargs) -> list[TableInfo]:
     return [
         TableInfo(tab) for tab in raw_table_info
     ]
+
+
+@rpc_method(name="tables.delete")
+@http_basic_auth_login_required
+@handle_rpc_exceptions
+def delete(
+    *, table_oid: int, cascade: bool = False, database_id: int, **kwargs
+) -> str:
+    """
+    Delete a table from a schema.
+
+    Args:
+        table_oid: Identity of the table in the user's database.
+        cascade: Whether to drop the dependent objects.
+        database_id: The Django id of the database containing the table.
```

Comment:
Ok, @mathemancer that's definitely different from how I tend to structure code. But I don't have a strong opinion on it. I'll defer to you. It might be nice to document a code standard about this at some point.